charts/postgres-db-sync/temp_table_sync.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+date;
+
```

Comment:
I would recommend to add 
```suggestion
set -e
```
ensuring that the script stops on the first error.
In the logs you showed on slack for example, if the create table file does not exist, we should not continue on and renaming tables in the final stage.